src/Components/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL/AspireEFPostgreSqlExtensions.cs

```diff
@@ -66,15 +66,108 @@ public static partial class AspireEFPostgreSqlExtensions
 
         configureSettings?.Invoke(settings);
 
-        if (settings.DbContextPooling)
+        builder.Services.AddNpgsqlDataSource(settings.ConnectionString ?? string.Empty, builder =>
```

Comment:
You are overwriting my changes in https://github.com/dotnet/aspire/commit/ae1163293782c49af36f84b5cc777bde5c218262. Can you revert this so it doesn't add back NpgsqlDataSource into DI?

Reply:
Done